NSAttributedStringFormat/NSAttributedString+CCLFormat.h

```diff
@@ -1,4 +1,8 @@
-@import Foundation;
+#if __has_include("NSAttributedString+CCLFormat-umbrella.h")
+@import Foundation; // dynamic framework
+#else
+#import <Foundation/Foundation.h>
+#endif
```

Comment:
I think this check is unnecessary and we can just `#import <Foundation/Foundation.h>` for all cases. The `__has_include` is tied to CocoaPods implementation details since it's detecting a file which CocoaPods decided to generate (could be moved/changed in future).

What do you think?